invenio_rdm_records/schemas/metadata.py

```diff
@@ -0,0 +1,506 @@
+        "ISTC",
+        "LISSN",
+        "LSID",
+        "PMID",
+        "PURL",
+        "UPC",
+        "URL",
+        "URN",
+        "w3id"
+    ]
+
+    identifier = SanitizedUnicode(required=True)
+    scheme = SanitizedUnicode(required=True, validate=validate.OneOf(
+            choices=SCHEMES,
+            error=_('Invalid related identifier scheme. ' +
+                    '{input} not one of {choices}.')
+        ))
+    relation_type = SanitizedUnicode(required=True, validate=validate.OneOf(
+            choices=RELATIONS,
+            error=_('Invalid relation type. {input} not one of {choices}.')
+        ))
+    resource_type = fields.Nested(ResourceTypeSchemaV1)
+
+
+class ReferenceSchemaV1(Schema):
+    """Reference schema."""
+
+    SCHEMES = [
+        "ISNI",
+        "GRID",
+        "Crossref Funder ID",
+        "Other"
+    ]
+    reference_string = SanitizedUnicode(required=True)
+    identifier = SanitizedUnicode()
+    scheme = SanitizedUnicode(validate=validate.OneOf(
+            choices=SCHEMES,
+            error=_('Invalid reference scheme. {input} not one of {choices}.')
+        ))
+
+
+class PointSchemaV1(Schema):
+    """Point schema."""
+
+    lat = fields.Number(required=True)
+    lon = fields.Number(required=True)
+
+
+class LocationSchemaV1(Schema):
+    """Location schema."""
+
+    point = fields.Nested(PointSchemaV1)
+    place = SanitizedUnicode(required=True)
+    description = SanitizedUnicode()
+
+
+class MetadataSchemaV1(Schema, FieldPermissionsMixin):
+    """Schema for the record metadata."""
+
+    field_load_permissions = {
+        # TODO: define "can_admin" action
+        # '_internal_notes': 'admin',
+    }
+
+    field_dump_permissions = {
+        # TODO: define "can_admin" action
+        # '_internal_notes': 'admin',
+    }
+
+    class Meta:
+        """Meta class to accept unknwon fields."""
+
+        unknown = INCLUDE
+
+    # Metadata fields
+    titles = fields.List(fields.Nested(TitleSchemaV1), required=True)
+    creators = fields.List(fields.Nested(CreatorSchemaV1), required=True)
+    resource_type = fields.Nested(ResourceTypeSchemaV1, required=True)
+    publication_date = EDTFLevel0DateString(required=True)
+    subjects = fields.List(fields.Nested(SubjectSchemaV1))
+    contributors = fields.List(fields.Nested(ContributorSchemaV1))
+    dates = fields.List(fields.Nested(DateSchemaV1))
+    language = SanitizedUnicode(validate=validate_iso639_3)
+    related_identifiers = fields.List(
+        fields.Nested(RelatedIdentifierSchemaV1))
+    version = SanitizedUnicode()
+    licenses = fields.List(fields.Nested(LicenseSchemaV1))
+    descriptions = fields.List(fields.Nested(DescriptionSchemaV1))
+    locations = fields.List(fields.Nested(LocationSchemaV1))
+    references = fields.List(fields.Nested(ReferenceSchemaV1))
+
+    _internal_notes = fields.List(fields.Nested(InternalNoteSchemaV1))
+
+    # TODO (Alex): this might go in a separate top-level field?
```

Comment:
What was meant here? We do need the extensions to the metadata schema in the metadata schema right?